src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -97,7 +97,7 @@ const logger = createLogger("Order/Routes/Shipping/index.tsx")
 @track()
 export class ShippingRoute extends Component<ShippingProps, ShippingState> {
   state: ShippingState = {
-    shippingOption: (this.props.order.requestedFulfillment &&
+    shippingOption: (this.props?.order?.requestedFulfillment &&
```

Comment:
shouldn't be optional

Reply:
Same here.

Otherwise I can't load the page and I get this lovely:

<img width="896" alt="Screen Shot 2021-02-26 at 3 32 07 PM" src="https://user-images.githubusercontent.com/10385964/109351765-ce0e8000-7847-11eb-974c-8eb752dcd253.png">


These failures aren't happening on staging, I've already confirmed. Seems to be an issue with my local `order` data.

Reply:
Is local data pointed at staging? Or is there some metaphysics WIP stuff happening? 

Again, adding `?` to stuff can be easy to overlook merge to master and if this just temporary we should add a `// FIXME: remove `?` before merge` (or something to that effect).